packages/rum/src/domain/record/shadowRootsController.ts

```diff
@@ -57,6 +57,8 @@ export const initShadowRootsController = (
         addTelemetryDebug('no shadow root in map', {
           shadowRoot: shadowRoot ? shadowRoot.nodeName : 'no node name',
           childrenLength: shadowRoot ? shadowRoot.childElementCount : '-1',
+          controllerByShadowRootSize: controllerByShadowRoot.size,
+          html: shadowRoot && isExperimentalFeatureEnabled('shadow_dom_debug') ? shadowRoot.innerHTML : undefined,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it could worth truncating the innerHTML to something like 1-2k to avoid sending huge payload